aws/cloudformation-templates/services/service/service.yaml

```diff
@@ -207,6 +215,9 @@ Resources:
           Image: !Ref ContainerImage
           Essential: true
           Memory: 256
+          Secrets:
+            - Name: OPTIMIZELY_SDK_KEY
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/retaildemostore-optimizely-sdk-key'
```

Comment:
Does this work with this SSM parameter being created and value set via CloudFormation template parameter? Didn't think secure strings could be created and initialized via CFN. 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html